src/v2/models/AppState.js

```diff
@@ -194,18 +194,15 @@ export default Model.extend({
       transformedResponse.currentAuthenticator?.poll?.refresh;
   },
 
-  // Sign Out link will be displayed in the footer of a form, unless
-  // - widget config hideSignOutLinkInMFA=true or mfaOnlyFlow=true
-  // and form is for identity verification (FORMS_FOR_VERIFICATION)
+  // Sign Out link will be displayed in the footer of a form, unless:
```

Comment:
it's unfortunate that the name of the parameter has `Sign Out` in it and now we call it `Back to sign in`.. can we at least mention that in comments and probably change the variable names ?

Reply:
Agree - we should come up with a deprecation strategy for fields that don't map the best with OIE enabled. I'll create a follow-up for a discussion on this.

Reply:
here in comment we can change to `Back to sign in formerly Sign Out link will be displayed .....`